pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -349,7 +349,9 @@ def make_package(args):
     if args.icon_fg and args.icon_bg:
         shutil.copy(args.icon_fg, join(res_dir, 'mipmap/icon_foreground.png'))
         shutil.copy(args.icon_bg, join(res_dir, 'mipmap/icon_background.png'))
-        with open(join(res_dir, 'mipmap-anydpi-v26/icon.xml'), "w") as fd:
+        anydpi_dir = join(res_dir, 'mipmap-anydpi-v26')
+        ensure_dir(anydpi_dir)
```

Comment:
`mipmap-anydpi-v26` is already present inside our template, and should be copied.

Can you better explain why that change is needed? Isn't the template working for your use-case?

Reply:
Hi @misl6. When you specify `--icon-fg` and `--icon-bg` for `--bootstrap=webview` you end up with error `FileNotFoundError: [Errno 2] No such file or directory: 'src/main/res/mipmap-anydpi-v26/icon.xml'`

Manually creating `mipmap-anydpi-v26` dir in `build.py` fixes this error. Could you, please, point where this dir is supposed to be created/copied?

Reply:
Exactly. This is explained in the commit message.

Reply:
[prepare_build_dir](https://github.com/kivy/python-for-android/blob/36b17aadff52b0b8d85bab5486c57e4151ffb523/pythonforandroid/bootstrap.py#L169) copies the content from the templates (which are placed here: [bootstraps](https://github.com/kivy/python-for-android/tree/develop/pythonforandroid/bootstraps))

> for --bootstrap=webview
Ouch, that makes sense.

The `mipmap-anydpi-v26` folder seems to only be available in `sdl2` boostrap: https://github.com/kivy/python-for-android/tree/develop/pythonforandroid/bootstraps/sdl2/build/src/main/res, but not in the `webview` one: https://github.com/kivy/python-for-android/tree/develop/pythonforandroid/bootstraps/webview/build/src/main/res

The logic should be cleaned up and unified, in order to avoid this kind of issues in future 🧐

Meanwhile (unless you would like to clean up and unify the logic), would be nice to add the missing folders to `webview`, in order to avoid extra checks in code.

Reply:
I've added the folder to the `bootstrap` template and removed the check code in the last commit.